.swiftlint.yml

```diff
@@ -49,9 +52,31 @@ type_name:
    warning: 80
    error: 100
 
+file_header:
+  required_pattern: |
+                    \/\/
+                    \/\/  .*?\.swift
+                    \/\/  .*?
```

Comment:
No way to make this compare against the actual file name (which is what the script was doing)?

Reply:
That fixed it :)